x-pack/plugins/actions/README.md

```diff
@@ -394,6 +398,45 @@ No parameters for the `incidentTypes` subaction. Provide an empty object `{}`.
 
 No parameters for the `severity` subaction. Provide an empty object `{}`.
 
+---
+## Swimlane
```

Comment:
This README Swimlane info looks good, but what about the rest of the [connector documentation](https://www.elastic.co/guide/en/kibana/master/action-types.html) - are you going to add it in the separate PR?

Reply:
I took a stab at the docs here: https://github.com/elastic/kibana/pull/100086/commits/b3e2ff6ec0588539131744309dde27090fb5b772

Is there a way to see them so that the images load 🤔  ?